src/parse/asp/parser.go

```diff
@@ -105,10 +105,11 @@ func (p *Parser) SubincludeTarget(state *core.BuildState, target *core.BuildTarg
 	}()
 	p.limiter.Acquire()
 	defer p.limiter.Release()
-	subincludePkgState := state
+	subincludePkgState := state.Root()
```

Comment:
This was causing a race because preloading from a subrepo was causing us to race on the RepoConfig field. We can guarantee that RepoConfig will be populated by now for the root repo and the subinclude target repo. The repo that this state object belongs to will most likely not be populated yet and will definitely race here.